src/connections/accounts.js

```diff
@@ -0,0 +1,66 @@
+import { withMutations } from 'cozy-client'
+
+const ACCOUNTS_DOCTYPE = 'io.cozy.accounts'
+const PERMISSIONS_DOCTYPE = 'io.cozy.permissions'
+
+const withAccountsMutations = withMutations(client => {
+  const createAccount = async (attributes, relationships) => {
+    const { data } = await client.create(
+      ACCOUNTS_DOCTYPE,
+      attributes,
+      relationships
+    )
+    return data
+  }
+
+  const findMasterAccount = async id => {
+    const { data } = await client.query(
+      client.find(ACCOUNTS_DOCTYPE).where({
+        _id: id
+      })
+    )
+    return data[0]
+  }
+
+  const createMasterAccount = async id => {
+    const { data } = await client.create(ACCOUNTS_DOCTYPE, {
+      _id: id
+    })
+    return data
+  }
+
+  const createSlaveAccount = async (konnector, attributes) => {
+    const masterAccountId =
+      konnector.aggregator && konnector.aggregator.accountId
```

Comment:
This line supposed `masterAccountId` could be undefined.
Is it not a problem for `values` l.46
and `_id` l.55 ?